src/client/cypress/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+      "target": "es5",
+      "lib": ["es5", "dom"],
+      "types": ["cypress"]
+    },
+    "include": ["**/*.ts", "integration/trades.js"]
```

Comment:
We wont want to maintain this list, the glob "**/*.ts" should be enough once we're using .ts files

Reply:
I made it. **/*.* to cover all kinds o ftest files